pykern/pkcli/__init__.py

```diff
@@ -162,6 +171,24 @@ def main(root_pkg, argv=None):
     return 0
 
 
+def _argh_name_mapping_policy():
+    """Set the argh name_mapping_policy.
+
+    In v0.30.0 argh changed the policy for mapping function args to
+    CLI args. For versions 0.30.0 and above this sets the policy to
+    BY_NAME_IF_HAS_DEFAULT which most closely matches the behavior
+    prior to v0.30.0.
+    """
+    from pykern.pkcollections import PKDict
```

Comment:
Do not import PKDict here. We try to avoid imports. I realize it can be imported, but it's better not, since this is before the pkcli has be initialized.

Reply:
7ff4012